docs/zh_cn/reference/how_to_setup_metadata_engine.md

```diff
@@ -57,6 +57,10 @@ $ juicefs format --storage s3 \
     pics
 ```
 
+:::caution 特别提示
+JuiceFS 对 redis 的最低版本要求为 4.0.0
```

Comment:
We should update `oldestSupportedVer` to 4.0  in `info.go`.

Also, update the comments in `redis.go`